test_tools/log_parser/parser.py

```diff
@@ -58,10 +58,14 @@ def cast(table, tmp_type):
     return tmp
 
 
-def parse_cb(payload):
+def parse_cb(payload, verbose):
     evt = CbEvent.GetRootAsCbEvent(payload, 0)
-    print(f'\tcb: actions:{evt.ActionIdsLength()} model:{evt.ModelId()} lm:{learning_mode_name(evt.LearningMode())} deferred:{evt.DeferredAction()}')
-    print(f'\t\tcontext: {fmt_payload(evt.ContextAsNumpy())}')
+    if not verbose:
+        print(f'\tcb: actions:{evt.ActionIdsLength()} model:{evt.ModelId()} lm:{learning_mode_name(evt.LearningMode())} deferred:{evt.DeferredAction()}')
+        print(f'\t\tcontext: {fmt_payload(evt.ContextAsNumpy())}')
```

Comment:
We probably don't want to print the context if !verbose, which is the biggest source of console spamming.